.gitignore

```diff
@@ -1,2 +1 @@
 node_modules
-package-lock.json
```

Comment:
Please remove this from the current PR. Please keep PRs down to a single thing for a better log.